tests/integration/live_data/zookeeper_tests.py

```diff
@@ -1,9 +1,15 @@
+import time
 import unittest
 
 from unittest import mock
 
+import gevent.socket
+
 try:
     from kazoo.exceptions import NoNodeError
+    from kazoo.handlers.gevent import SequentialGeventHandler
+    from kazoo.handlers.threading import SequentialThreadingHandler
+    from kazoo.retry import KazooRetry
 except ImportError:
```

Comment:
Out of the scope of this PR since you didn't change it, but this seems like something that shouldn't be skipped if something is not installed, especially for tests.